indexer/src/types/interfaces.ts

```diff
@@ -0,0 +1,64 @@
+// Starknet
+import { Transaction, TransactionReceipt } from "../deps.ts";
+
+// Eth
+import {
+  JsonRpcTx,
+  PrefixedHexString,
+  TypedTransaction,
+  JsonTx,
+} from "../deps.ts";
+
+/**
+ * Represents a hexadecimal string with a `0x` prefix.
+ */
+export type HexString = `0x${string}`
+
+/**
+ * Represents a request to convert a transaction to Ethereum transaction format.
+ */
+export interface ToEthTxRequest {
+  transaction: Transaction; //  A Ethereum transaction.
+  receipt: TransactionReceipt; // The Ethereum receipt corresponding to a reverted out of resources transaction.
+  blockNumber: PrefixedHexString; // The block number in which the transaction was included, as a prefixed hex string.
+  blockHash: PrefixedHexString; // The hash of the block in which the transaction was included, as a prefixed hex string.
+  isPendingBlock: boolean; // Indicates if the transaction is in a pending block.
```

Comment:
I meant the documentation header which is displayed correctly above the field in question for example:

```ts
export interface ToEthTxRequest {
/** An Ethereum transaction. */
 transaction: Transaction;
```